libc/src/fenv/CMakeLists.txt

```diff
@@ -5,7 +5,8 @@ add_entrypoint_object(
   HDRS
     fegetround.h
   DEPENDS
-    libc.include.fenv
+    libc.hdr.fenv_t
+    libc.hdr.fexcept_t
```

Comment:
Remove both of these dependency, they are not directly used in `fegetround.(h|cpp)`.